macros/schema_tests/json_paths_not_null.sql

```diff
@@ -0,0 +1,22 @@
+{% macro test_json_paths_not_null (model) %}
+    {% set column_name = kwargs.get('column_name') %}
+    {% set paths = kwargs.get('paths', []) %}
+
+    with validation_errors as (
+        select
+            {% for path in paths %}
```

Comment:
As above, I think it might be better to get rid of this `for` loop, as it will make this test easier to debug if it fails